### PR TITLE
Create ManualBreadcrumbs and use for task directory

### PR DESCRIPTION
**Pull request by kennyt**
px

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @orlandohohmeier

File: src/js/components/Breadcrumbs.js [hunk 1]
```diff
@@ -1,186 +1,18 @@
...
 
-import BreadcrumbSegmentLink from './BreadcrumbSegmentLink';
-import IconChevron from './icons/IconChevron';
-
-const COLLAPSE_BUFFER = 12;
-const LAST_ITEM_OFFSET = 150; // Difference between scrollWidth and outerWidth
-const PADDED_ICON_WIDTH = 38; // Width of icon + padding
+import ManualBreadcrumbs from './ManualBreadcrumbs';
 
 class Breadcrumbs extends mixin(StoreMixin) {
```
Comment (kennyt) [inline src/js/components/Breadcrumbs.js]: i'd like to rename this to `RouteBreadcrumbs` but then that can probably go inside of a separate PR

File: src/js/components/ManualBreadcrumbs.js [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+import ReactDOM from 'react-dom';
+
+import BreadcrumbSegmentLink from './BreadcrumbSegmentLink';
+import IconChevron from './icons/IconChevron';
+
+const COLLAPSE_BUFFER = 12;
+const LAST_ITEM_OFFSET = 150; // Difference between scrollWidth and outerWidth
+const PADDED_ICON_WIDTH = 38; // Width of icon + padding
+
+class ManualBreadcrumbs extends React.Component {
```
Comment (jfurrow) [inline src/js/components/ManualBreadcrumbs.js]: I'm not crazy about the word `Manual` but I can't think of anything better...

Comment (orlandohohmeier) [inline src/js/components/ManualBreadcrumbs.js]: I'd like rename this one to just `Breadcrumbs` and the other to `RoutedBreadcrumbs`  as this one will serve as the base class for all breadcrumbs. What do you think?

File: src/js/components/Breadcrumbs.js (hunk 1 above)
Comment (orlandohohmeier) [inline src/js/components/Breadcrumbs.js]: https://github.com/dcos/dcos-ui/pull/501/files#r68375170

**Comment by pierlo-upitup** [comment]
Thanks @kennyt this is a great step in the right direction. Next, let's make sure we roll out a plan to reduce the number of available `*Breadcrumbs` components out there and we'll live happily thereafter.

File: src/js/components/ManualBreadcrumbs.js (hunk 2 above)
Comment (kennyt) [inline src/js/components/ManualBreadcrumbs.js]: yep, i agree
